src/main/java/ch/naviqore/raptor/router/RaptorRouter.java

```diff
@@ -131,7 +131,7 @@ private List<Query.Label[]> spawnFromStops(int[] sourceStopIndices, int[] target
         // initially relax all source stops and add the newly improved stops by relaxation to the marked stops
         Set<Integer> markedStops = query.initialize();
         markedStops.addAll(footpathRelaxer.relaxInitial(sourceStopIndices));
-        markedStops = query.removeSubOptimalLabelsForRound(0, markedStops);
```

Comment:
Although not orthographically correct, camel casing in our company treats prefixes (sub, super, pre, post, etc.) as independent words. I generally prefer this because it improves readability but know that it's strictly speaking incorrect.